rest/model/image.go

```diff
@@ -59,6 +59,76 @@ func (apiToolchain *APIToolchain) ToService() *thirdparty.Toolchain {
 	return &toolchain
 }
 
+////////////////////////////////////////////////////////////////////////////////
+//
+// APIImageEventEntry is the model to be returned by the API whenever image event entries are fetched.
+
+type APIImageEventEntry struct {
+	Name   *string                          `json:"name"`
+	Before *string                          `json:"before"`
+	After  *string                          `json:"after"`
+	Type   thirdparty.ImageEventType        `json:"image_event_type"`
```

Comment:
any reason why this isn't just `type`?
```suggestion
	Type   thirdparty.ImageEventType        `json:"type"`
```